README.md

```diff
@@ -1,3 +1,71 @@
-# action-artefacts-ci
+## How to use it:
 
-WIP
\ No newline at end of file
+### 1. **Add the Action to your Workflow**
+
+In your GitHub workflow file (usually located in .github/workflows/), you can add the `artefacts-ci` action to a job. Here's an example:
+    
+```
+jobs:
+  artefacts-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: art-e-fact/action-artefacts-ci@main
+        with:
+          artefacts-api-key: ${{ secrets.ARTEFACTS_API_KEY }}
+          job-name: basic_tests
+```
+
+In this example, replace `basic_tests` with the name of your job that you already have in your `artefacts.yaml` file.
+
+Optionally, you can add a [`ros test`](https://github.com/ros-tooling/setup-ros) job before the artefacts-ci to ensure everything is working before running the platform job:
+    
+```
+jobs:
+  ros2-test:
+    runs-on: ubuntu-latest
+    env:
+      ROS_DISTRO: humble
+    steps:
+        - uses: ros-tooling/setup-ros@v0.6
```

Comment:
I guess a checkout would be needed here as well?

Reply:
not sure about dolly, but here, either we add the checkout, or we skip this section of the README

Reply:
@deku-95 Can you add the checkout step so that we can merge the PR

Reply:
Done